scripts/devops_tasks/test_regression.py

```diff
@@ -32,8 +31,9 @@
     clone_repo,
 )
 
-from ci_tools.functions import discover_targeted_packages, str_to_bool, find_whl
+from ci_tools.functions import discover_targeted_packages, find_whl, parse_require
```

Comment:
Was `parse_require` supposed to be moved over as well?

Reply:
Yep!

Reply:
But I needed to pull this one from variables, so your callout is right. It's wrong